src/modules/SdnDiag.Server.psm1

```diff
@@ -2569,16 +2569,19 @@ function Get-SdnVMNetworkAdapter {
         Import-Module -Name Hyper-V -Force -ErrorAction Stop
     }
 
-    # remove the credential parameter if it is empty to avoid passing it to the cmdlet
+    $vmNetworkAdaptersParams = $PSBoundParameters
```

Comment:
nitpick: if Get-SdnVMNetworkAdapter is exported and not just an internal function, it might be good to use parameter sets to perform some validation (e.g., so `Get-SdnVMNetworkAdapter -All -VMName 5b8fbfb6-MUX0 -Name HNVPA` won't error out)

Reply:
Thanks. I will fix in https://github.com/microsoft/SdnDiagnostics/issues/435